### DISCUSSION TITLE
White Page with optional catch-all for the root segment

### DISCUSSION THREAD

**Discussion by Dev-rdg** [vercel/next.js#50649] [Help]
### Summary

Hello!

I'm using [[...uid]].tsx as the root page for my website. This page fetches content from prismic to statically render pages including the homepage.

Everything works fine when i run npm run dev or npm run start after building the app.
But as soon as i deploy to vercel i get a white page when navigating from any page to the homepage.
To be clear, the homepage works on the first load but any click on a link with href="/" will show me the white page.
No errors whatsoever in the browser console or in vercel logs

Can someone help?

Thanks in advance

**Comment by devjiwonchoi** [comment]
I have something coming up in my mind, but before that, what does it look like in your `Source`?

You can view it by `Overview > "Your Project" > Deployments > "Your Recent Deployment" > "View Source" > Source > Output`

![Screenshot 2023-06-02 at 3 06 09 AM](https://github.com/vercel/next.js/assets/120007119/0dd5ad0c-d29d-4c5a-bbed-cd158c5cc6e9)

**Comment by Dev-rdg** [comment]
There it is.
________________________________
De : 최지원 ***@***.***>
Envoyé : jeudi 1 juin 2023 20:08
À : vercel/next.js ***@***.***>
Cc : Lucas S ***@***.***>; Author ***@***.***>
Objet : Re: [vercel/next.js] White Page with optional catch-all for the root segment (Discussion #50649)


I have something coming up in my mind, but before that, what does it look like in your Source?

You can view it by Overview > "Your Project" > Deployments > "Your Recent Deployment" > "View Source" > Source > Output

[Screenshot 2023-06-02 at 3 06 09 AM]<https://user-images.githubusercontent.com/120007119/242686991-0dd5ad0c-d29d-4c5a-bbed-cd158c5cc6e9.png>

**Reply by devjiwonchoi** [reply]
I meant in your Vercel Deployment. Maybe you've pasted the wrong one by mistake?

**Comment by Dev-rdg** [comment]
I went to : Deployment > clicked on the production deployment's view Source > Output tab

isn't that correct ?
________________________________
De : 최지원 ***@***.***>
Envoyé : vendredi 2 juin 2023 10:58
À : vercel/next.js ***@***.***>
Cc : Lucas S ***@***.***>; Author ***@***.***>
Objet : Re: [vercel/next.js] White Page with optional catch-all for the root segment (Discussion #50649)


I meant in your Vercel Deployment. Maybe you've pasted the wrong one by mistake?

**Reply by devjiwonchoi** [reply]
Yes that's correct. Could you show me the output?

**Comment by Dev-rdg** [comment]
Is this it?
When i click Invoke Serverless Function i get the second screenshot
________________________________
De : 최지원 ***@***.***>
Envoyé : vendredi 2 juin 2023 11:30
À : vercel/next.js ***@***.***>
Cc : Lucas S ***@***.***>; Author ***@***.***>
Objet : Re: [vercel/next.js] White Page with optional catch-all for the root segment (Discussion #50649)


Yes that's correct. Could you show me the output?

**Reply by devjiwonchoi** [reply]
![Screenshot 2023-06-02 at 6 43 06 PM](https://github.com/vercel/next.js/assets/120007119/65051078-46c8-4aab-a6cc-16cdf66e4319)
This is what I see right now.

**Reply by Dev-rdg** [reply]
My bad I was replying with my email client and github seems to be stripping the attachments!
Is this what you were looking for?
Thanks you for your patience!

![Screenshot_20230602_113350](https://github.com/vercel/next.js/assets/80473041/3cf95529-a8b0-48df-986d-84b2cf0f4eba)
![Screenshot_20230602_113220](https://github.com/vercel/next.js/assets/80473041/8b7458cb-1e29-48b2-9405-00e56615eeb2)
![Screenshot_20230602_113129](https://github.com/vercel/next.js/assets/80473041/42ca5d1a-0910-440f-9d71-9cec23fcf93c)

**Reply by devjiwonchoi** [reply]
Have you tried setting your `Link` component `href` to `[[...uid]]`?

Example:

```js
<Link href="[[...uid]]" />
```
